R/trackplot.R

```diff
@@ -2364,12 +2381,16 @@ summarize_homer_annots = function(anno, sample_names = NULL, legend_font_size =
   mat
 }
 
-.extract_summary = function(bw, binSize, bed, op_dir){
+.extract_summary = function(bw, binSize, bed, op_dir,use_conda = NULL){
   bn = gsub(pattern = "\\.bw$|\\.bigWig$", replacement = "",
             x = basename(bw), ignore.case = TRUE)
   message(paste0("    Processing ", bn, ".."))
-  
-  cmd = paste("bwtool summary -with-sum -keep-bed -header", bedSimple, bw, paste0(op_dir, bn, ".summary"))
+  if (!is.null(use_conda)){
+    cmd_conda = paste0("conda run -n ", use_conda)
```

Comment:
Hi!
Indeed, the `use_conda` argument should be designed to accept the name of the conda environment. It is not appropriate to simply name it `use_conda`, as this could potentially mislead users. I suggest renaming it to `use_conda_env` to avoid confusion and to clearly communicate the purpose of the argument.
I believe that bwtool is much easier to install via conda. I think this improvement would make this package more robust.